packages/flutter/lib/src/material/theme.dart

```diff
@@ -161,6 +161,48 @@ class Theme extends StatelessWidget {
         .resolveFrom(context);
   }
 
+  /// Retrieves the [Brightness] to use for descendant Material widgets, based
+  /// on the value of [ThemeData.brightness] in the given [context].
+  ///
+  /// If no [InheritedTheme] can be found in the given [context], or its `brightness`
+  /// is null, it will fall back to [MediaQueryData.platformBrightness].
+  ///
+  /// Throws an exception if no valid [InheritedTheme] or [MediaQuery] widgets
```

Comment:
```suggestion
```

I had a look at the implementation of https://api.flutter.dev/flutter/widgets/MediaQuery/platformBrightnessOf.html

and it returns `Brightness.light` instead of throwing if there is no MediaQuery.
So you can remove the phrase that says it throws.